docker/test/sqltest/run.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# shellcheck disable=SC2015
+
+set -x
+set -e
+set -u
+set -o pipefail
+
+BINARY_TO_DOWNLOAD=${BINARY_TO_DOWNLOAD:="clang-16_debug_none_unsplitted_disable_False_binary"}
```

Comment:
Please, don't use the worst practices from performance tests. Binaries should be downloaded in advance and mounted into the container by a script from `tests/ci`

Reply:
Ok. I use copy-paste and ChatGPT to write Python. And when I'm using copy-paste, I'm trying to reuse the shortest code, so I have less amount to read.